lld/ELF/Driver.cpp

```diff
@@ -2956,9 +2998,12 @@ void LinkerDriver::link(opt::InputArgList &args) {
   // partition.
   mainPart = &partitions[0];
 
-  // Read .note.gnu.property sections from input object files which
-  // contain a hint to tweak linker's and loader's behaviors.
-  config->andFeatures = getAndFeatures();
+  // Read:
+  // - .note.gnu.property sections from input object files which
+  //   contain a hint to tweak linker's and loader's behaviors;
+  // - .note.AARCH64-PAUTH-ABI-tag sections contents from input object files
```

Comment:
Since the ABI has been updated to say that the GNU program property scheme is preferred and I don't think compatibility with .note.AARCH64-PAUTH-ABI-tag is needed, we can just drop support for .note.AARCH64-PAUTH-ABI-tag

Reply:
Dropped support for the alternative and implemented support for GNU property. See 978ed06259a3b6ea035dde2c07581f62c45cfd4a